specs/lib/obj2gltfSpec.js

```diff
@@ -184,4 +184,17 @@ describe('obj2gltf', function() {
             obj2gltf(texturedObjPath, options);
         }).toThrowDeveloperError();
     });
+
+    it('adds KHR_materials_unlit to materials if unlit is set', function(done) {
```

Comment:
I don't think this test is needed since it is already covered in `adds KHR_materials_unlit extension when unlit is set`